lib/server.js

```diff
@@ -65,9 +34,61 @@ internals.serveDefaultPage = function serveDefaultPage (res, args) {
   src = src.replace('JS', args.js.alias)
   src = src.replace('CSS', args.css.alias)
 
-  res.setHeader('Content-Type', 'text/html')
+  return src
+}
+
+internals.customDefaultHTML = ''
+
+internals.setCustomDefaultHTML = function setCustomDefaultHTML (file, args){
+  var html = fs.readFileSync(file, {encoding: 'utf8'})
+    , shouldReplaceJS = ~html.indexOf(JS_REPLACE)
+    , shouldReplaceCSS = ~html.indexOf(CSS_REPLACE)
```

Comment:
Oh crap, sorry. That's probably not good in production code. It's hard to read. 

`~` is a bit shift operator. so…

```js
~-1 === 0
```

It's just a shortcut for `html.indexOf(JS_REPLACE) > -1`